akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1239,8 +1239,8 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    */
   @Deprecated
   @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.12")
-  def groupedWithin(n: Int, d: FiniteDuration): javadsl.Flow[In, java.util.List[Out], Mat] =
-    new Flow(delegate.groupedWithin(n, d).map(_.asJava)) // TODO optimize to one step
+  def groupedWithin(n: Int, duration: FiniteDuration): javadsl.Flow[In, java.util.List[Out], Mat] =
```

Comment:
Shall I change `n` -> `maxNumber`?

Reply:
Sounds good

Reply:
Done in 8fe645ff01655413589ac9778ab2205652eac49a

I did not touch scaladsl classes because it is a breaking change